BootloaderCorePkg/Tools/CfgDataStitch.py

```diff
@@ -87,8 +87,8 @@ def cfgdata_stitch(ifwi_file, ifwi_out_file, cfg_dir, key_file, script_dir, tool
             (script_dir, ['GenCfgData.py', 'CfgDataTool.py']),
             (cfg_dir, ['CfgDataDef.dsc']),
         ]
-    # Check for KEY_ID at end of string
-    if not key_file.endswith('KEY_ID'):
+    # Check for KEY_ID in key file string
+    if not 'KEY_ID' in key_file:

```

Comment:
do you have specific rules for the KEY_ID name conversion ?  
will '_KEY_ID_' be better ?

Reply:
Now, we have defined as **COMPONENT**_KEY_ID_**SIGNTYPESIZE**. Prior it this patch it use end with KEY_ID.
Yes, KEY_ID_**COMPONENT**_**SIGNTYPESIZE** would be better, Will update these namings.